llama-index-integrations/llms/llama-index-llms-gemini/llama_index/llms/gemini/base.py

```diff
@@ -89,7 +89,7 @@ class Gemini(CustomLLM):
     def __init__(
         self,
         api_key: Optional[str] = None,
-        model_name: Optional[str] = GEMINI_MODELS[0],
+        model: Optional[str] = GEMINI_MODELS[0],
```

Comment:
To avoid breaking changes, we should keep the old kwarg, and mark as deprecated for now

Reply:
yes my bad, i've made a commit to correct this is this ok for you ?